### PR TITLE
refactor(core/modal): Improve wizardPage types so no type param is necessary

### PR DESCRIPTION
**Pull request by christopherthielen**
Added some T's

### CODE REVIEW

File: app/scripts/modules/core/src/modal/wizard/WizardModal.tsx [hunk 1]
```diff
@@ -13,10 +13,10 @@ import { SubmitButton } from '../buttons/SubmitButton';
 
 import { IWizardPageProps, IWizardPageValidate } from './WizardPage';
 
-export interface IWizardPageData {
+export interface IWizardPageData<T> {
```
Comment (christopherthielen) [inline app/scripts/modules/core/src/modal/wizard/WizardModal.tsx]: added some `T`s

**Comment by christopherthielen** [comment]
@danielpeach @sbwsg @pivotal-Jammy-Louie FYI

File: app/scripts/modules/amazon/src/loadBalancer/configure/common/SecurityGroups.tsx [hunk 2]
```diff
@@ -193,7 +193,11 @@ class SecurityGroupsImpl extends React.Component<ISecurityGroupsProps, ISecurity
                     The following {FirewallLabels.get('firewalls')} could not be found in the selected
                     account/region/VPC and were removed:
                   </p>
-                  <ul>{removed.map(sg => <li key={sg}>{sg}</li>)}</ul>
+                  <ul>
+                    {removed.map(sg => (
+                      <li key={sg}>{sg}</li>
+                    ))}
+                  </ul>
```
Comment (jrsquared) [inline app/scripts/modules/amazon/src/loadBalancer/configure/common/SecurityGroups.tsx]: stop it prettier.

Comment (christopherthielen) [reply app/scripts/modules/amazon/src/loadBalancer/configure/common/SecurityGroups.tsx]: Is it possible your prettier is out of date?  I'm going to re-run current prettier across the entire codebase next.

**Pull request merged**